R/Rules-methods.R

```diff
@@ -326,6 +326,192 @@ setMethod("nextBest",
           })
 
 
+##' --------------------------------------------------
+##' NCRM method with loss function & overdose control
+##' --------------------------------------------------
+##' @describeIn nextBest Find the next best dose based on the NCRM method and
+##' loss function.
+##' @importFrom ggplot2 ggplot geom_bar xlab ylab ylim aes geom_vline
+##' geom_hline geom_point
+##' @importFrom gridExtra arrangeGrob
```

Comment:
```suggestion
```
not needed anymore, as we import whole ggplot2 and also the other helper see https://github.com/Roche/crmPack/blob/9835f38e30a546c28d36438caf49cee251d7e753/R/crmPack-package.R#L31

Reply:
removed